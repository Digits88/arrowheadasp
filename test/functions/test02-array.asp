<!-- Test ARRAY function.
  Input is a comma-delimited list of initial values.
-->

Dim A <%Dim A%><br>
A = Array("a", "b", "c", "d") <%A = Array("a", "b", "c", "d")%><br>
A(0) = <%=A(0)%><br>
A(1) = <%=A(1)%><br>
A(2) = <%=A(2)%><br>
A(3) = <%=A(3)%><br>
