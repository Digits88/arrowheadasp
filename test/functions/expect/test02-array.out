<!-- Test ARRAY function.
  Input is a comma-delimited list of initial values.
-->

Dim A <br>
A = Array("a", "b", "c", "d") <br>
A(0) = a<br>
A(1) = b<br>
A(2) = c<br>
A(3) = d<br>
